woocommerce/wc-admin
# Remove wc-api customers

After #1325 was merged, I think we don't need these files anymore.

### Detailed test instructions:
- Go to the _Customers_ report.
- Verify there are no regressions and data is loading fine.
